WhiteDentalClinic/WhiteDentalClinic.Api/Controllers/AppointmentController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System.ComponentModel.DataAnnotations;
+using WhiteDentalClinic.Application.Models;
+using WhiteDentalClinic.Application.Models.AppointmentModel;
+using WhiteDentalClinic.Application.Services.Interfaces;
+
+namespace WhiteDentalClinic.Api.Controllers
+{
+    public class AppointmentController : ApiController
```

Comment:
The controller name should be plural -> AppointmentsController so that the route will be something like "api/appointments/..." based on what you specified by using the [Route("api/[controller]")] attribute on the BaseController. (Applicable to all controllers.)